static/scripts/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+info_color='\033[0;34m'
+
+info_msg()
+{
+    message=$1
+    echo -e "$info_color[info]:: $message $default_color"
+    echo ''
+}
+
+info_msg "Clonning flojoy-ai/studio repo..."
+
+git clone --branch main --recurse-submodules https://github.com/flojoy-ai/studio.git studio
+
+cd studio
+
+info_msg "Creating a python virtual environment..."
+
+python3 -m venv venv
+
+if [ $? -ne 0 ]; then
+    python -m venv venv
+fi
+
+info_msg "Activating virtual environment..."
+
+source venv/bin/activate
```

Comment:
I think we should skip the activation step, because after u activate, it will stay unless u deactivate. Users may not know what they are doing and accidentally re-use this venv for other stuff. Which can lead to things breaking